test/e2e_node/runner/remote/run_remote.go

```diff
@@ -701,6 +704,12 @@ func parseInstanceMetadata(str string) map[string]string {
 		}
 		metadata[kp[0]] = string(v)
 	}
+	envs := strings.TrimSpace(*nodeEnvs)
+	if len(envs) > 0 {
+		for _, e := range strings.Split(envs, ",") {
+			metadata[e] = os.Getenv(e)
```

Comment:
Is it possible to also pass the env value in the flag? Feel a little big strange to read env in this file.

Reply:
We can, but we need to handle escaping `,`.
Or we need to support multiple instances of the same flag like we do for `extra-log`.

I feel like this seems simpler.

Reply:
Yeah, worth adding an info log for the envs? 

LGTM.

Reply:
Addressed, change it to multiple flags.

Reply:
There is already a log `parsed instance metadata`. :)